.github/workflows/golangci-lint.yml

```diff
@@ -16,5 +16,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.29
```

Comment:
This version doesn't support `go@1.19`. I tried using `setup-go` to pin an old version of Go, but that didn't work, so I updated the linter